### ISSUE TITLE
BBA did not repond correct to stayman

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#707]
0A2409ECE15A4C50AC52B86C078D
![image](https://github.com/user-attachments/assets/8e388b83-2349-46f7-a3fe-33baa68d6642)

Perhaps it is the other error, where BBA want to bid 3C with 5H that is causing the problem.
Hepefully we get both fixed.

**Comment by EdwardPiwowar** [comment]
W can't bid 2C as multi-landy.

**Comment by ThorvaldAagaard** [comment]
Then double can't be stayman :-)
I think it is better to respond to Stayman if it is Stayman. I assume Multi-Landy can be 5-4 so we can still play 4Ma

**Comment by EdwardPiwowar** [comment]
Next BBA, double can't be stayman :-)
![image](https://github.com/user-attachments/assets/7f0e4f11-2cfe-4e38-9feb-d60662c8e31f)

**Comment by ThorvaldAagaard** [comment]
So what is double, 2D, 2H, 2S, 2N, 3C, 3D, 3H, 3S?
Normally Lebensohl is 2N followed by 3N to show the stoppers, where 3N directly denies stoppers.

Remember we have normally system on after 2C overcall, now you are introducing a totally new area of bidding.

**Comment by EdwardPiwowar** [comment]
If after 1N-(2C)-? you use stayman and transfers, you don't use Lebensohl, and vice versa.

**Comment by ThorvaldAagaard** [comment]
Agree, but you just changed it for Multi-Landy because it was showing majors.

So what is 
1N-(2C)-2H after Multi-Landy?

You should have only 2 version after opening 1N and overcall. Either system is or, or you play Lebensohl,
